test/integration/targets/vmware_local_role_facts/tasks/main.yml

```diff
@@ -42,11 +42,7 @@
   assert:
     that:
       - "{{ fact_details.changed == false }}"
-      - "{{ fact_details.local_role_facts['Admin']['privileges'] is defined }}"
-      - "{{ fact_details.local_role_facts['Admin']['role_id'] is defined }}"
-      - "{{ fact_details.local_role_facts['Admin']['role_info_label'] is defined }}"
-      - "{{ fact_details.local_role_facts['Admin']['role_info_summary'] is defined }}"
-      - "{{ fact_details.local_role_facts['Admin']['role_system'] is defined }}"
+      - fact_details.local_role_facts is defined
```

Comment:
Instead of this test case, can we have something - 

```yaml
  vars:
    role_list:
      - Admin
      - NoCryptoAdmin
      - NoAccess
      - Anonymous
      - ReadOnly
  hosts: localhost
  tasks:
    - name: Gather Role facts
      vmware_local_role_facts:
        hostname: "{{ vcenter_server }}"
        username: "{{ vcenter_user }}"
        password: "{{ vcenter_pass }}"
        validate_certs: no
      register: role_details

    - name: Test if role id is present for role
      assert:
        that: "{{ role_details.local_role_facts | json_query(s_query) != [] }}"
      vars:
        s_query: "[?role_name == '{{ item }}'].role_id"
      with_items: "{{ role_list }}"
```